extras/create_send_notifications.py

```diff
@@ -107,12 +121,12 @@ def main():
     os.chdir(os.path.join(NOTIFICATIONS_BASE_DIR, NOTIFICATION_ARCHIVE_DIR))
 
     # Build list of CyHy orgs
-    cyhy_org_ids = build_cyhy_org_list(db)
-    logging.debug("Found {} CYHY orgs: {}".format(len(cyhy_org_ids), cyhy_org_ids))
+    notification_org_ids = build_notifications_org_list(db)
+    logging.debug("Found {} CYHY orgs: {}".format(len(notification_org_ids), notification_org_ids))
```

Comment:
Update debug verbiage:
```suggestion
    logging.debug("Will attempt to generate notifications for {} orgs: {}".format(len(notification_org_ids), notification_org_ids))
```